lighthouse-core/driver/index.js

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
This file and web-inspector didn't get detected as renames.  Usually happens when >50% of the file changes. Would be great to keep git history if possible, but nbd.